tabs/configuration.js

```diff
@@ -446,6 +451,104 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
         $('input[name="currentoffset"]').val(BF_CONFIG.currentoffset);
         $('input[name="multiwiicurrentoutput"]').prop('checked', MISC.multiwiicurrentoutput);
 
+        var escProtocols = {
```

Comment:
@digitalentity @stronnag needs some help with allowed rates and defaults. Specially for default for BRUSHED since never had anything to do with them